### PR TITLE
feat(bmd): Check passcodes for admin cards

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview
The useInsertedSmartcardHook requires checking the admin card's pin as of #2031 

Also remove GLOBALS.CARD_POLLING_INTERVAL in favor of the constant from the hook.

## Demo Video or Screenshot

https://user-images.githubusercontent.com/530106/176321511-45b4f1ca-0670-405e-9223-4a056d11e876.mov

## Testing Plan 
- Update existing unit tests
- Some manual testing

## Checklist
- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [ ] I have added JSDoc comments to any newly introduced exports

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: frontends/bmd/src/app_mark_voter_card_invalid.test.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ describe('Mark Card Void when voter is idle too long', () => {
     // User action removes Idle Screen
     fireEvent.click(screen.getByText('Yes, I’m still voting.'));
     fireEvent.mouseDown(document);
-    await advanceTimersAndPromises();
+    await advanceTimersAndPromises(0.2);
```
Comment (arsalansufi) [inline frontends/bmd/src/app_mark_voter_card_invalid.test.tsx:73]: Why was this change needed?

Comment (jonahkagan) [reply frontends/bmd/src/app_mark_voter_card_invalid.test.tsx:73]: Oh yeah I meant to leave a PR comment explaining this. There's a `setTimeout` with a delay of 200ms that triggers the closing of the idle screen after user activity. 200ms happened to be the value of BMD's `GLOBALS.CARD_POLLING_INTERVAL` before I replaced it with the card polling interval from the auth hook (which is only 100ms). `advanceTimersAndPromises` defaults to advancing timers by the card polling interval, so this was coincidentally "just working" before.
